samples/web/send-from-aspnetcore-webapi/Core_6/WebApplication/SendMessageController.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using NServiceBus;
+
+[Route("api/[controller]")]
+public class SendMessageController : Controller
+{
+    IMessageSession messageSession;
+
+    #region MessageSessionInjection
+    public SendMessageController(IMessageSession messageSession)
+    {
+        this.messageSession = messageSession;
+    }
+    #endregion
+
+
+    #region MessageSessionUsage
+    [HttpGet]
+    public async Task<string> Get()
```

Comment:
I know this is only a sample but sending messages in a `HTTP GET` feels wrong...